README.rst

```diff
@@ -30,6 +40,60 @@ Xarray Extended with Climate Data Analysis Tools
 .. |Checked with mypy| image:: http://www.mypy-lang.org/static/mypy_badge.svg
    :target: http://mypy-lang.org/
 
+
+XCDAT is a Python library built on `xarray`_ for climate data analysis on structured grids.
+It serves as a spiritual successor to the Community Data Analysis Tools (`CDAT`_) library with a focus on long-term maintainability and extensibility.
+
+.. _xarray: https://github.com/pydata/xarray
+.. _CDAT: https://github.com/CDAT/cdat
+
+Planned Features
+-----------------
+
+XCDAT will provide climate domain specific features and utilities that are helpful for general xarray usage.
+
+These features include:
+
+- Support for metadata that is CF compliant, E3SM non-CF compliant, and common
+- Robust handling of coordinates and its associated bounds
+
+  - Name-agnostic retrieval of CF compliant coordinates and bounds using ``cf_xarray``
+  - Generating specific bounds or filling all missing bounds for supported axes
+  - Ability to operate on both (0 to 360) and (-180 to 180) longitudinal axes orientations
+
+- Temporal averaging (weighted or unweighted)
+
+  - Time series averaging and calculation of climatologies and anomalies
+  - Use of time bounds for calculating weights
+  - Optional centering of time using time bounds
+
+- Geospatial weighted averaging over rectilinear grid
+
+  - Optional specification of regional domain
+
+- Horizontal and vertical structured regridding
+
+  - Operate on rectilinear and cuvilinear grids
+
+- Wrappers for opening datasets to apply common operations
+
+  - Fill missing bounds
+  - Decoding of CF and non-CF time units
+  - Optional centering of time axes using time bounds
+  - Optional conversion of longitudinal axes orientation
+
+- Support for parallelism of XCDAT features using Dask
```

Comment:
This section is intended to be a high-level summary of the planned features. More or less detail can be provided.